pontos/version/version.py

```diff
@@ -47,6 +50,7 @@ def _print(self, *args) -> None:
     def get_current_version(self) -> str:
         """Get the current version of this project"""
         # implementme.
+        return ""
```

Comment:
IMHO it is better to raise an exception here
```suggestion
        raise NotImplementedError() ""
```